scene/gui/tree.cpp

```diff
@@ -1654,6 +1652,25 @@ void Tree::_update_theme_item_cache() {
 	theme_cache.base_scale = get_theme_default_base_scale();
 }
 
+Size2 Tree::_get_cell_icon_size(const TreeItem::Cell &p_cell) const {
+	Size2i icon_size = p_cell.get_icon_size();
+
+	int max_width = 0;
+	if (theme_cache.icon_max_width > 0) {
+		max_width = theme_cache.icon_max_width;
+	}
+	if (p_cell.icon_max_w > 0 && p_cell.icon_max_w < max_width) {
```

Comment:
I didn't test this one, but likely same.

Reply:
I figured you were just reading the code, no worries :)